api/v1/db/oracledb/connection-example.js

```diff
@@ -1,10 +1,11 @@
 const config = require('config');
 const oracledb = require('oracledb');
 
-process.on('SIGINT', () => process.exit());
+const dbConfig = config.get('database');
 
+process.on('SIGINT', () => process.exit());
 oracledb.outFormat = oracledb.OBJECT;
-const dbConfig = config.get('database');
+oracledb.fetchAsString = [oracledb.DATE, oracledb.NUMBER];
```

Comment:
This line is to fix the issue mentioned in [here](https://oracle.github.io/node-oracledb/doc/api.html#-13162-fetching-numbers).